.github/workflows/windows-build.yml

```diff
@@ -85,10 +85,12 @@ jobs:
       run: |
         set PATH=%PATH:C:\Program Files\Git\bin;=%
         set PATH=%PATH:C:\Program Files\Git\usr\bin;=%
+        bcdedit /set IncreaseUserVa 3072
```

Comment:
Is this the maximum limit we can set ?

Reply:
`IncreaseUserVa` needs to be in 2048 ~ 3072 on https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/bcdedit--set
But I also see this command is only available on 32-bit OS on some comments.